src/ops/mersenne-rand.ts

```diff
@@ -0,0 +1,129 @@
+/**
```

Comment:
Hi sorry for the delay. Working with some open source folks at Google to okay this file. Just to check -- where is this implementation taken from?

Reply:
So unfortunately we can't accept the copy paste of this code like this due to licensing issues -- is it possible to reimplement this algorithm using the source C++ simply as a reference?

Reply:
this file is deleted, but somehow still showing here, I think reviewable is displaying it deleted